### PR TITLE
New Velocloud Edge Metrics

### PR DESCRIPTION
**Pull request by Mamoon-Khan**
Added new metric in VeloCloud SNMP profile

### CODE REVIEW

**Comment by thezackm** [comment]
thank you for your submission @Mamoon-Khan 

please update this PR to change the individual OIDs listed from the scalar functions they are now into a table configuration for the [vceLinkTable](https://mibs.observium.org/mib/VELOCLOUD-EDGE-MIB/#vceLinkTable) that these all look to be members of.

you can see an example of how to pivot these into a table configuration with `metric_tags` here: https://github.com/kentik/snmp-profiles/blob/ccb1df47a5068a59fb3e3765746524e0286252e7/profiles/kentik_snmp/_template.yml#L41-L78

File: Velo-Static-Circuits-2023-10-31.csv
Comment (thezackm) [inline Velo-Static-Circuits-2023-10-31.csv:1]: please remove this file

**Review by thezackm — CHANGES_REQUESTED** [review]
please redact the CSV file from this PR

File: Velo-Static-Circuits-2023-10-31.csv
Comment (Mamoon-Khan) [reply Velo-Static-Circuits-2023-10-31.csv:1]: done

File: profiles/kentik_snmp/velocloud/velocloud-edge.yml [hunk 1]
```diff
@@ -32,3 +32,29 @@ metrics:
...
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.25
+      - name: VEC Link State
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.34
+    metric_tags:
+      - name: Velocloud Edge Link Name
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.3
+      - name: VCE Path Peer Name
+        OID: 1.3.6.1.4.1.45346.1.1.2.4.2.2.1.6
+      - name: VCE Link Itf
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.33
```
Comment (thezackm) [inline profiles/kentik_snmp/velocloud/velocloud-edge.yml:60]: this is an enumerated OID; as it sits right now the only value returned will be an integer; you'll want to add the enumeration of `VceLinkStateType` from [the MIB file](https://jira.observium.org/secure/attachment/19423/VeloCloudEdge510-EnterpriseMib.txt) to make this human-readable.

```yaml
metric_tags:
  - tag: link_state
    column:
      name: vceLinkState
      oid: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.34
      enum:
        initial: 1
        dead: 2
        unusable: 3
        quiet: 4
        standby: 5
        unstable: 6
        stable: 7
        unknown: 8
```

**Review by thezackm — CHANGES_REQUESTED** [review]
a few more edits; looks a lot better

**Review by thezackm — APPROVED** [review]
🚀  thanks for the submission!

**Pull request merged**